src/mono/mono/mini/mini-amd64.c

```diff
@@ -2368,6 +2400,9 @@ mono_arch_get_llvm_call_info (MonoCompile *cfg, MonoMethodSignature *sig)
 		linfo->ret.storage = LLVMArgVtypeRetAddr;
 		linfo->vret_arg_index = cinfo->vret_arg_index;
 		break;
+	case ArgSwiftValuetypeLoweredRet:
+		linfo->ret.storage = LLVMArgNone; // LLVM compilation of pinvoke wrappers is not supported, see emit_method_inner in mini-llvm.c
```

Comment:
Should it assert here?

Reply:
No, we can't assert here. See https://github.com/dotnet/runtime/pull/104949#discussion_r1681069779.